cli/cmd/setup_filesystem_config.go

```
@@ -0,0 +1,88 @@
+package cmd
+
+import (
+	"fmt"
+	"log/slog"
+
+	"github.com/spf13/cobra"
+
+	"ocm.software/open-component-model/bindings/go/configuration/filesystem/v1alpha1"
+	v1 "ocm.software/open-component-model/bindings/go/configuration/v1"
+	"ocm.software/open-component-model/bindings/go/runtime"
+	ocmctx "ocm.software/open-component-model/cli/internal/context"
+)
+
+// setupFilesystemConfig sets up file system configuration entity.
+func setupFilesystemConfig(cmd *cobra.Command) {
+	value, _ := cmd.PersistentFlags().GetString(tempFolderFlag)
```

why do you ignore the error here?

---

Thanks. I wanted to fix that.